### ISSUE TITLE
Scipy.Signal

### ISSUE THREAD

**Issue by natank1** [SciSharp/NumSharp#349]
Hi

Is there a way to run spectogram with scipy.signal in Using this library?

**Comment by Nucs** [comment]
No, Unfortunately we do not have implementations of [scipy.signal](https://docs.scipy.org/doc/scipy/reference/signal.html).
I would suggest you to use [Numpy.NET](https://github.com/SciSharp/Numpy.NET) as it wraps numpy directly and provides all its features.

**Comment by natank1** [comment]
Including SciPy ? 

נשלח מה-iPhone שלי

‫ב-21 בספט׳ 2019, בשעה 12:38, ‏‏Eli Belash ‏<notifications@github.com> כתב/ה:‬

**Comment by Nucs** [comment]
Scipy is the name of the company that developed numpy...

**Comment by natank1** [comment]
Thanks

נשלח מה-iPhone שלי

‫ב-21 בספט׳ 2019, בשעה 13:43, ‏‏Eli Belash ‏<notifications@github.com> כתב/ה:‬

**Comment by natank1** [comment]
Sorry again

When you say the Scipy exists you mean we have an API for it under
Numpy.Net (I can find such ),
or we can take the python code of scipy and arrange it in away that
Numpy.net will handle this?
Sorry again fro bothering

**Comment by Nucs** [comment]
Scipy is the name of the company that published Numpy. When you say Scipy it makes no sense.
When I comment I add URLs, just click on my mentions of numpy.net.
Numpy.NET is a different library from NumSharp which uses Pythonnet to call numpy.
Numpy.NET implements ALL numpy's functions but might be sometimes slower because you transfer data from C# to python.

**Comment by natank1** [comment]
So function /class that is written in python sipy.signal , how should be called inumpy.net?
נשלח מה-iPhone שלי

‫ב-21 בספט׳ 2019, בשעה 17:06, ‏‏Eli Belash ‏<notifications@github.com> כתב/ה:‬

**Comment by Nucs** [comment]
I have no clue, This is NumSharp repository.
Read Numpy.NET's readme.md.

**Comment by natank1** [comment]
Thanks!
